projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1237,6 +1441,58 @@
 
   "biome.quark.glimmering_weald": "微光荒野",
 
+  "tag.item.c.storage_blocks.apple": "苹果存储方块",
+  "tag.item.c.storage_blocks.bamboo": "竹存储方块",
+  "tag.item.c.storage_blocks.beetroot": "甜菜根存储方块",
+  "tag.item.c.storage_blocks.blaze_rod": "烈焰棒存储方块",
+  "tag.item.c.storage_blocks.cactus": "仙人掌存储方块",
+  "tag.item.c.storage_blocks.carrot": "胡萝卜存储方块",
+  "tag.item.c.storage_blocks.charcoal": "木炭存储方块",
+  "tag.item.c.storage_blocks.chorus_fruit": "紫颂果存储方块",
+  "tag.item.c.storage_blocks.cocoa_beans": "可可豆存储方块",
+  "tag.item.c.storage_blocks.glow_berries": "发光浆果存储方块",
+  "tag.item.c.storage_blocks.golden_apple": "金苹果存储方块",
+  "tag.item.c.storage_blocks.golden_carrot": "金胡萝卜存储方块",
+  "tag.item.c.storage_blocks.gunpowder": "火药存储方块",
+  "tag.item.c.storage_blocks.leather": "皮革存储方块",
+  "tag.item.c.storage_blocks.nether_wart": "下界疣存储方块",
+  "tag.item.c.storage_blocks.potato": "马铃薯存储方块",
+  "tag.item.c.storage_blocks.rabbit_hide": "兔子皮存储方块",
+  "tag.item.c.storage_blocks.stick": "木棍存储方块",
+  "tag.item.c.storage_blocks.sugar_cane": "甘蔗存储方块",
+  "tag.item.c.storage_blocks.sweet_berries": "甜浆果存储方块",
+
+  "tag.item.quark.ancient_logs": "远古木",
+  "tag.item.quark.azalea_logs": "杜鹃木",
+  "tag.item.quark.bamboo_logs": "竹",
+  "tag.item.quark.big_harvesting_hoes": "范围收获锄",
```

有点别扭，“能范围收获的锄”比较容易一眼看懂

---

其实我写出来的时候，脑子里在播放哆啦A梦拿出道具的场景，不过你说的这个好像也很搭w
```suggestion
  "tag.item.quark.big_harvesting_hoes": "可以范围收获的锄",
```